[DRAFT] Add So_Long

vbrabandt opened the pull request:
a simple 2D game created with [MiniLibX](https://github.com/42Paris/minilibx-linux) with [@Grumpyx3](https://github.com/Grumpyx3)

Features:
- Player character turns depending on direction
- Exit opens and closes

Bugs:
- Norminette Errors

vbrabandt committed:
First draft

vbrabandt committed:
Initial commit

- Work from 2 September 2024

Development goon into stand still, though we did progress a bit before.

We implemented Improved movement and Player Character turning depending on direction.

But right now we have minor memory leaks (which we don't know where it is and the 42-Norminette Line count is out of place

But yeah I still have hopes....

(P.S. Fuck Norminette)

vbrabandt committed:
Update README.md

vbrabandt committed:
Draft kinda

vbrabandt committed:
Renaming and Makefile improvements

vbrabandt committed:
Partial Norminette fix

vbrabandt committed:
Update start_game.c

vbrabandt committed:
Update README.md

vbrabandt committed:
Update README.md

vbrabandt committed:
Update README.md

vbrabandt2005 commented:
Needs new assets
![capybara](https://github.com/user-attachments/assets/97ed09ba-80bc-4d2e-9cc4-e82df4627a09)

vbrabandt committed:
fclean

vbrabandt committed:
Replaced assets

- Replaced assets with nice Capybara assets
- Replaced something with cute hand-drawn Pelican
- Fixed Makefile not fcleaning properly

vbrabandt committed:
Some Improvements

- Increased assets size to 120
- Changed Closed exit to not be hidden
- added enemies in maps/ex03.ber

vbrabandt committed:
Removed snprintf from move_count.c

Since we're not allowed to use anything from the printf group (except for our own ft_printf), I had to remove snprintf in favour of something else.

The only things which are left now are Norminette errors (hopefully......)

vbrabandt committed:
Update Makefile

vbrabandt committed:
Update move_count.c

vbrabandt committed:
Update README.md

vbrabandt committed:
Update README.md

vbrabandt committed:
Update README.md

vbrabandt committed:
fixed some norminette errors, ended creating more

vbrabandt committed:
partial norminette fix

vbrabandt committed:
Almost Norminette fix, one error left

vbrabandt committed:
Everything completed!!!!

AHHHHHHHHHHHHHHHHH

vbrabandt committed:
Update end_game.c

vbrabandt committed:
stuff

vbrabandt committed:
Improved choose_map.sh and added new map

vbrabandt committed:
Update so_long.h
